src/models/data-cube/filters.ts

```diff
@@ -1,35 +1,27 @@
-import { Category, Filter, FilterOptions, Row } from './types';
+import { Category, Filter, RangeOptions, Row } from './types';
 
-const millisecondsPerDay = 24 * 60 * 60 * 1000;
-
-const defaultOptions: FilterOptions = {
-  excludeStartDate: false,
-  excludeEndDate: false,
-};
+export function betweenDates(startDate: Date, endDate: Date, rangeOptions?: RangeOptions): Filter {
+  const dateRange: [Date, Date] = [startDate, endDate];
+  return inOneOfDateRanges([dateRange], rangeOptions);
+}
 
-export function betweenDates(startDate: Date, endDate: Date, options: Partial<FilterOptions> = {}): Filter {
-  const { excludeStartDate, excludeEndDate } = {
-    ...defaultOptions,
-    ...options,
-  };
+export function inOneOfDateRanges(dateRanges: [Date, Date][], rangeOptions: RangeOptions = {}): Filter {
+  const {
+    excludeStart = false,
+    excludeEnd = false,
+  } = rangeOptions;
+  const startOffset = excludeStart ? 1 : 0;
```

Comment:
Could we use a constant for the offset value "1"? To make it clear what time increment it is.